lasso-marko-plugin.js

```diff
@@ -95,18 +95,23 @@ module.exports = function(lasso, config) {
                 });
             } else {
                 if (meta.deps) {
-                    dependencies = dependencies.concat(meta.deps.map(depPath => ({
-                        type: 'require',
-                        path: require.resolve(this.resolvePath(depPath, nodePath.dirname(this.path)))
-                    })));
+                    dependencies = dependencies.concat(meta.deps.map(dep => (
+                        dep.code ? {
+                            type: dep.type,
+                            code: dep.code 
+                        } : {
+                            type: dep.includes(':') ? dep.slice(0, dep.indexOf(':')) : 'require',
```

Comment:
Also is it possible that someone may have registered a custom Lasso dependency type that has a colon in it? For example `mycustom:dependency`. In this case, I think that this would cause problems. I highly doubt that this would be an issue, but I think technically this could cause a problem.